### PR TITLE
Fix settings/mentorship SyntaxError

### PR DESCRIPTION
**Pull request by mariocsee**
## What type of PR is this? (check all applicable)
- [x] Bug Fix

## Description
Remove string addition that caused syntax error.

## Related Tickets & Documents
#1908 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
N/A

## Added to documentation?
- [x] no documentation needed

### CODE REVIEW

File: app/views/users/_mentorship.html.erb [hunk 1]
```diff
@@ -26,8 +26,7 @@
     <p>Please share the top 3 technologies/concepts you'd like to work on with your mentor, how much experience you currently have (0-10, 0 being no experience at all), and why you're interested in learning more. Please also share your general technical background and career goals.</p>
     <%= f.text_area(
           :mentee_description,
-          placeholder: "For example:\n\n1. React - 1, I walked through the tutorial on the React website, but I'm feeling lost. I see React listed in lots of job descriptions so I'd like to learn how to utilize the framework.\n\n"
-            + "I graduated from a coding bootcamp 3 months ago and my goal is to land my first developer role. I learned jQuery in the program.",
+          placeholder: "For example:\n\n1. React - 1, I walked through the tutorial on the React website, but I'm feeling lost. I see React listed in lots of job descriptions so I'd like to learn how to utilize the framework.\n\n I graduated from a coding bootcamp 3 months ago and my goal is to land my first developer role. I learned jQuery in the program.",
```
Comment (Glennmen) [inline app/views/users/_mentorship.html.erb]: This should have failed pre-commit linting check
```
Metrics/LineLength: Line is too long. [360/289] (https://github.com/rubocop-hq/ruby-style-guide#80-character-limits)
In file: app/views/users/_mentorship.html.erb:33
```

Comment (mariocsee) [reply app/views/users/_mentorship.html.erb]: I ran `bundle exec erblint app/views/users/_mentorship.html.erb` and it showed me that error, but when I made changes and committed I didn't get that message.

Comment (mariocsee) [reply app/views/users/_mentorship.html.erb]: Anyways, just made the necessary changes and `/settings/mentorships` doesn't hit a SyntaxError. Thanks for the heads up!

Comment (Glennmen) [reply app/views/users/_mentorship.html.erb]: 👍

**Pull request merged**